models/user/user.go

```diff
@@ -355,6 +353,44 @@ func GetUserFollowing(ctx context.Context, u, viewer *User, listOptions db.ListO
 	return users, count, err
 }
 
+// GetUserFollowersCount returns count of user's followers.
+func GetUserFollowersCount(ctx context.Context, u, viewer *User, listOptions db.ListOptions) (int64, error) {
+	sess := db.GetEngine(ctx).
+		Select("`user`.*").
+		Join("LEFT", "follow", "`user`.id=follow.user_id").
+		Where("follow.follow_id=?", u.ID).
+		And("`user`.type=?", UserTypeIndividual).
+		And(isUserVisibleToViewerCond(viewer))
+
+	if listOptions.Page != 0 {
+		sess = db.SetSessionPagination(sess, &listOptions)
+		count, err := sess.Count()
+		return count, err
+	}
+
+	count, err := sess.Count()
+	return count, err
+}
+
```

Comment:
```suggestion
```

Reply:
maybe only remove L367 and L368 is enough?